ui/src/components/view/SearchView.vue

```diff
@@ -347,6 +348,14 @@ export default {
         this.fillFormFieldValues()
       })
     },
+    sortArray (key, data) {
```

Comment:
```suggestion
    sortArray (data, key='name') {
```
Minor nit, just to make life easier for newer devs